Clients/Xamarin.Interactive.Client/WorkbookAppInstallation.cs

```diff
@@ -363,5 +365,28 @@ IEnumerable<string> Pathify (string path)
                 Pathify (appJson.GetValue ("ticketProviderAssembly")?.Value<string> ())?.SingleOrDefault (),
                 order);
         }
+
+        static string RidPathMapper (string path)
+        {
+            const string ridVar = "{rid}";
+            var index = path.IndexOf (ridVar, StringComparison.Ordinal);
+            if (index < 0)
+                return path;
+
+            var versionsParentDir = path.Substring (0, index);
```

Comment:
Why do you need to care about this?

Seems to me this entire method could just be hoisted to `pathMappers`:

```csharp
... {
    path => path.Replace ("{rid}", Runtime.CurrentProcessRuntime.RuntimeIdentifier);
}
```

Reply:
If you look at the path mapper for the XM workbook apps, it searches in the parent dir for the closest match to the host system. I was thinking we'd do the same here, except with RIDs.

But we're not really doing that, so yeah, a simple replace as you suggest would be fine here.

I'll need to backport your Runtime stuff to 1.4.